EXPERIMENTAL: disable the default input grabbing; at least while testing
and debugging

Index: rts/Game/UnsyncedGameCommands.cpp
--- rts/Game/UnsyncedGameCommands.cpp.orig
+++ rts/Game/UnsyncedGameCommands.cpp
@@ -3963,7 +3963,7 @@ void UnsyncedGameCommands::AddDefaultActionExecutors()
 	AddActionExecutor(AllocActionExecutor<IncreaseGUIOpacityActionExecutor>());
 	AddActionExecutor(AllocActionExecutor<DecreaseGUIOpacityActionExecutor>());
 	AddActionExecutor(AllocActionExecutor<ScreenShotActionExecutor>());
-	AddActionExecutor(AllocActionExecutor<GrabInputActionExecutor>());
+	//AddActionExecutor(AllocActionExecutor<GrabInputActionExecutor>());
 	AddActionExecutor(AllocActionExecutor<ClockActionExecutor>());
 	AddActionExecutor(AllocActionExecutor<CrossActionExecutor>());
 	AddActionExecutor(AllocActionExecutor<FPSActionExecutor>());
